app/views/layouts/_header.html.slim

```diff
@@ -3,9 +3,9 @@
   - if publisher_signed_in?
     = header.navigation_item text: t("nav.manage_jobs"), href: organisation_path, active: manage_jobs_active?
     - if current_organisation.school_group?
-      = header.navigation_item text: t("nav.manage_settings"), href: publishers_schools_path, active: schools_in_your_trust_active?
+      = header.navigation_item text: t("nav.organisation_profile"), href: publishers_schools_path, active: schools_in_your_trust_active?
     - else
-      = header.navigation_item text: t("nav.manage_settings"), href: publishers_school_path(current_organisation), active: schools_in_your_trust_active?
+      = header.navigation_item text: t("nav.school_profile"), href: publishers_school_path(current_organisation), active: schools_in_your_trust_active?
```

Comment:
In my experience, we consider both Schools and SchoolGroups as Organisations. Perhaps a bit nit picky, but perhaps `nav.school_group_profile` is better or something like that?

Reply:
Yeah, I can see the language is a bit ambiguous here. The way I see it here is that school is a specialised type of organisation, so when there is only one organisation a user can interact with, then the link says School profile. Otherwise, it says Organisation profile. I am not certain if school_group would work here, as it also covers LAs.

Reply:
I _think_ we consider LAs to be "school groups" as well.

Reply:
But I guess if it says "Organisation profile", organisation is fine 😅

Reply:
I'll mark this as resolved for now, but we definitely need more conversation around the language. :)